gradle/libs.versions.toml

```diff
@@ -1,7 +1,7 @@
 [versions]
 
-nva = { strictly = '1.13.1' }
-nvaDatamodel = { strictly = '0.13.11' }
+nva = { strictly = '1.14.0' }
```

Comment:
this is only a coincidence

Reply:
Ha, you say "coincidence"…let me remind you of 197…